packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -35,6 +35,10 @@ export class CXLMarketingNavElement extends LitElement {
   @property({ type: Boolean, reflect: true })
   minimal = false;
 
+  // Conditionally hide search box
+  @property({ type: Boolean, reflect: true })
+  hideSearch = false;
```

Comment:
Configurations towards the negative are mentally taxing.

Refactor to `showSearch` instead.